lib/components/app/app-menu.js

```diff
@@ -1,25 +1,35 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import qs from 'qs'
 import { connect } from 'react-redux'
-import { DropdownButton, MenuItem } from 'react-bootstrap'
 import { withRouter } from 'react-router'
+import SlidingPane from 'react-sliding-pane'
+import { MenuItem } from 'react-bootstrap'
+import VelocityTransitionGroup from 'velocity-react/velocity-transition-group'
+import { injectIntl } from 'react-intl'
```

Comment:
Please sort imports

Reply:
Sorted in c870fb55202566b78808a5b6e9f61009a9860d24!